llvm/test/CodeGen/PowerPC/vcmp-setbc-quad.ll

```diff
@@ -0,0 +1,15 @@
+; RUN: llc -mcpu=pwr10 -mtriple=powerpc64-ibm-aix -ppc-asm-full-reg-names \
+; RUN:     -ppc-vsr-nums-as-vr < %s | FileCheck %s
+; RUN: llc -mcpu=pwr10 -mtriple=powerpc64le-unknown-linux-gnu \
+; RUN:     -ppc-asm-full-reg-names -ppc-vsr-nums-as-vr < %s | FileCheck %s
```

Comment:
Does it also make sense to add a `-mtriple=powerpc-ibm-aix` RUN line like you did in the other test?

Reply:
The compiler doesn't support __int128 in 32-bit mode.

Reply:
Oops, you're right. Thanks for the reminder, Roland. Please disregard this comment.